packages/squiggle-lang/src/value/valueTags.ts

```diff
@@ -45,6 +52,19 @@ function convertToValueTagsTypeName(
 export class ValueTags {
   constructor(public value: ValueTagsType) {}
 
+  exportData(): VDict | undefined {
```

Comment:
This is slightly inconsistent with other accessors in this class (which return plain JS values), but it shouldn't matter if we implement my recent proposal about storing all tags as Value instances.